stdlib/types.pyi

```diff
@@ -123,7 +123,31 @@ class CodeType:
             freevars: Tuple[str, ...] = ...,
             cellvars: Tuple[str, ...] = ...,
         ) -> None: ...
-    if sys.version_info >= (3, 8):
+    if sys.version_info >= (3, 10):
+        def replace(
+            self,
+            *,
+            co_argcount: int = ...,
+            co_posonlyargcount: int = ...,
+            co_kwonlyargcount: int = ...,
+            co_nlocals: int = ...,
+            co_stacksize: int = ...,
+            co_flags: int = ...,
+            co_firstlineno: int = ...,
+            co_code: bytes = ...,
+            co_consts: Tuple[Any, ...] = ...,
+            co_names: Tuple[str, ...] = ...,
+            co_varnames: Tuple[str, ...] = ...,
+            co_freevars: Tuple[str, ...] = ...,
+            co_cellvars: Tuple[str, ...] = ...,
+            co_filename: str = ...,
+            co_name: str = ...,
+            co_lnotab: bytes = ...,
```

Comment:
According to stubtest, `lnotab` is gone on 3.10 (see last line of output below). There are several other stubtest errors too, but it would be best to fix them in a separate PR.

```
(env310) akuli@akuli-desktop:~/typeshed$ python3 -m mypy.stubtest --concise --custom-typeshed-dir=. types | grep replace
types.CodeType.replace is inconsistent, runtime argument "co_cellvars" has a default value of type None, which is incompatible with stub argument type builtins.tuple[builtins.str]
types.CodeType.replace is inconsistent, runtime argument "co_code" has a default value of type None, which is incompatible with stub argument type builtins.bytes
types.CodeType.replace is inconsistent, runtime argument "co_consts" has a default value of type None, which is incompatible with stub argument type builtins.tuple[Any]
types.CodeType.replace is inconsistent, runtime argument "co_filename" has a default value of type None, which is incompatible with stub argument type builtins.str
types.CodeType.replace is inconsistent, runtime argument "co_freevars" has a default value of type None, which is incompatible with stub argument type builtins.tuple[builtins.str]
types.CodeType.replace is inconsistent, runtime argument "co_name" has a default value of type None, which is incompatible with stub argument type builtins.str
types.CodeType.replace is inconsistent, runtime argument "co_names" has a default value of type None, which is incompatible with stub argument type builtins.tuple[builtins.str]
types.CodeType.replace is inconsistent, runtime argument "co_varnames" has a default value of type None, which is incompatible with stub argument type builtins.tuple[builtins.str]
types.CodeType.replace is inconsistent, runtime does not have argument "co_lnotab"
```

```suggestion
```

Reply:
Actually, none of these can actually be `None`, even though stubtest thinks so.